node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
`tl.loc(__dirname, 'resourceKey', formatArg1, formatArg2)` breaks compat. The current interface expects the resource key to be first and there is no normal way to distinguish whether you really meant __dirname instead of resourceKey.

However, there is an unorthodox way to distinguish. If the first arg is a cache miss, then check if it's a directory...